cloudify-manager-worker/templates/statefulset.yaml

```diff
@@ -71,7 +71,11 @@ spec:
         - name: cloudify-cfg-volume
           mountPath: /tmp/cloudify/config.yaml
           subPath: config.yaml
-        {{- if .Values.licence }}
+        {{- if .Values.license }}
+        - name: cloudify-license-volume
+          mountPath: /tmp/cloudify/cfy_license.yaml
+          subPath: cfy_license.yaml
+        {{- else if .Values.licence }}
```

Comment:
please be consistent -> license instead of licence

Reply:
That's in order to comply with old environments upgrades in the future, changing it in one hit will cause future upgrades/restart to fail upload the license as it'll look for license in the statefulSet but have cfy_licence.yaml in the configMap itself.
There's also the possibility of an upgrade and not updating the values file, or reusing the current one - which will cause more issues